src/main/java/mate/academy/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -101,22 +106,24 @@ public Order delete(Long id) {
     @Override
     public List<Order> getOrders(Long userId) {
         List<Order> list = new ArrayList<>();
-        try (Statement statementOrder = connection.createStatement()) {
-            String queryOrders = String.format("SELECT * FROM orders WHERE user_id = %d;",
-                    userId);
-            ResultSet resultSet = statementOrder.executeQuery(queryOrders);
+        String queryOrders = "SELECT * FROM orders WHERE user_id = ?;";
+        try (PreparedStatement statementOrder = connection.prepareStatement(queryOrders)) {
+            statementOrder.setLong(1, userId);
+            ResultSet resultSet = statementOrder.executeQuery();
             while (resultSet.next()) {
                 Long orderId = resultSet.getLong("order_id");
                 Order order = new Order(userId);
                 order.setId(orderId);
                 List<Item> listItem = new ArrayList<>();
-                try (Statement statementItems = connection.createStatement()) {
-                    String queryItems = String.format(
-                            "SELECT * FROM items i INNER JOIN orders_items oi "
-                            + "ON i.item_id = oi.item_id "
-                                    + "INNER JOIN orders o ON oi.order_id = o.order_id "
-                            + "WHERE o.user_id = %d and o.order_id = %d;", userId, orderId);
-                    ResultSet resultSetItems = statementItems.executeQuery(queryItems);
+                String queryItems =
```

Comment:
maybe better divide this method into two parts? second part will have name getAllItemsFromOrder(Long orderId, Long userId)?

Reply:
OK.